<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRecipeGrabberUi</class>
 <widget class="QWidget" name="QtRecipeGrabberUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPlainTextEdit" name="textEdit">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="titleButton">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ingredientsButton">
       <property name="text">
        <string>Ingredients</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="instructionsButton">
       <property name="text">
        <string>Instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sourceButton">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="servingsButton">
       <property name="text">
        <string>Servings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>291</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleButton</tabstop>
  <tabstop>ingredientsButton</tabstop>
  <tabstop>instructionsButton</tabstop>
  <tabstop>sourceButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
